modules/nw-sriov-operator-uninstall.adoc

```diff
@@ -66,21 +66,11 @@ $ oc delete crd sriovnetworks.sriovnetwork.openshift.io
 $ oc delete crd sriovoperatorconfigs.sriovnetwork.openshift.io
 ----
 
-. Delete the SR-IOV webhooks:
+. Change the SR-IOV Network Operator configuration custom resource (CR):
```

Comment:
Is this change approved by the QA?
They should test this before we change this in the documentation